### ISSUE TITLE
Why are dictionaries disallowed for attribute types?

### ISSUE THREAD

**Issue by saschanaz** [heycam/webidl#938]
9e764b7d197e46fdf6fcf2da11e9aa0315445b28 disallowed it but with no background. Does anyone have an idea why?

(Found it from https://github.com/w3c/deviceorientation/pull/90)

**Comment by domenic** [comment]
Yes, because they are copied as sequences are, so you'd fail to have x.attr === x.attr.

**Comment by saschanaz** [comment]
Thanks!

Currently [the sequence section](https://heycam.github.io/webidl/#idl-sequence) says it's disallowed for attributes but [the dictionary section](https://heycam.github.io/webidl/#idl-dictionary) does not, so the work item here would be add a corresponding prose with notes.
